src/elements/common/annotator-context/useAnnotatorEvents.ts

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { EventEmitter } from 'events';
+import { AnnotationActionEvent, Status } from './types';
+
+export interface UseAnnotatorEventsProps {
+    eventEmitter: EventEmitter;
+    onAnnotationDeleteEnd?: (annotationId: string) => void;
+    onAnnotationDeleteStart?: (annotationId: string) => void;
+    onAnnotationUpdateEnd?: (annotation: Object) => void;
+    onAnnotationUpdateStart?: (annotation: Object) => void;
+    onSidebarAnnotationSelected?: (annotationId: string) => void;
+}
+
+const emitUpdateAnnotationEvent = (eventEmitter: EventEmitter, annotation: Object, status: Status) => {
```

Comment:
Should this be declared inside the hook to stay consistent with the others? That way, it doesn't need `eventEmitter` passed to it.